src/my_project/main.py

```
@@ -326,11 +315,14 @@ def search(self, text: str):
             self.library_search(text)
 
     def library_search(self, text: str):
-        display_info = text_search(text)
+        with FTS5Searcher(config_manager) as searcher:
+            display_info = searcher.text_search(text)
         if display_info is None:
             # TODO: Need to add logic here.
             raise ValueError("Incorrect type passed!")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)